hikari/audit_logs.py

```diff
@@ -74,7 +74,6 @@ class AuditLogChangeKey(str, enums.Enum):
     ICON_HASH = "icon_hash"
     SPLASH_HASH = "splash_hash"
     OWNER_ID = "owner_id"
-    REGION = "region"  # TODO: remove when this disappears for the most part
```

Comment:
This removal feels a bit pre-mature with the whole audit-logs persist for like 3 months and all thing